src/Cassandra.IntegrationTests/Core/ExceptionsTests.cs

```diff
@@ -110,28 +139,29 @@ public void NoHostAvailableException()
         [Test]
         public void ReadTimeoutException()
         {
-            ITestCluster nonShareableTestCluster = TestClusterManager.GetNonShareableTestCluster(2);
-            ISession session = nonShareableTestCluster.Session;
-
-            string keyspace = "TestKeyspace_" + Randomm.RandomAlphaNum(10);
-            string table = "TestTable_" + Randomm.RandomAlphaNum(10);
-            int replicationFactor = 2;
-            string key = "1";
-
-            session.Execute(String.Format(TestUtils.CreateKeyspaceSimpleFormat, keyspace, replicationFactor));
-            Thread.Sleep(5000);
-            session.ChangeKeyspace(keyspace);
-            session.Execute(String.Format(TestUtils.CreateTableSimpleFormat, table));
-            Thread.Sleep(3000);
-
-            session.Execute(
-                new SimpleStatement(String.Format(TestUtils.INSERT_FORMAT, table, key, "foo", 42, 24.03f)).SetConsistencyLevel(
-                    ConsistencyLevel.All));
-            session.Execute(new SimpleStatement(String.Format(TestUtils.SELECT_ALL_FORMAT, table)).SetConsistencyLevel(ConsistencyLevel.All));
-
-            nonShareableTestCluster.StopForce(2);
-            var ex = Assert.Throws<ReadTimeoutException>(() => 
-                session.Execute(new SimpleStatement(String.Format(TestUtils.SELECT_ALL_FORMAT, table)).SetConsistencyLevel(ConsistencyLevel.All)));
+            var cql = string.Format(TestUtils.SELECT_ALL_FORMAT, _table);
+            var primeQuery = new
+            {
+                when = new
+                {
+                    query = cql
+                },
+                then = new
+                {
+                    result = "read_timeout",
+                    consistency_level = 5,
+                    received = 1,
+                    block_for = 2,
+                    data_present = true,
+                    delay_in_ms = 0,
+                    message = "read_timeout",
+                    ignore_on_prepare = false
+                }
+            };
+
+            _simulacronCluster.Prime(primeQuery);
+            var ex = Assert.Throws<ReadTimeoutException>(() =>
+                _session.Execute(new SimpleStatement(cql).SetConsistencyLevel(ConsistencyLevel.All)));
```

Comment:
As we are dealing with errors, I think it would be best to create a new `Cluster` instance each time and dispose them at the end.